cargo-workspaces/src/publish.rs

```diff
@@ -54,6 +54,17 @@ pub struct Publish {
     /// Don't remove dev-dependencies while publishing
     #[clap(long)]
     no_remove_dev_deps: bool,
+
+    /// Only print the order in which crates should be published.
+    #[clap(long)]
+    only_print_plan: bool,
```

Comment:
Can you please make this (plan printing) a separate subcommand (`plan`)?

Reply:
Done. Also extracted some common code to utils to prevent code duplication.